src/mapping/nav2_map_server/test/integration/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+include_directories( ${PROJECT_SOURCE_DIR}/test )
+
+ament_add_gtest_executable(test_occ_grid_node
+  test_occ_grid_node.cpp
+  ${PROJECT_SOURCE_DIR}/test/test_constants.cpp
+)
+ament_target_dependencies(test_occ_grid_node rclcpp nav_msgs)
+target_link_libraries(test_occ_grid_node
+  map_reps
+  ${ament_LIBRARIES}
```

Comment:
`${ament_LIBRARIES}` is not needed. It is already set as a dependency as part of the `ament_target_dependencies` call